src/viur/core/modules/user.py

```diff
@@ -286,11 +285,20 @@ def getAuthMethodName(*args, **kwargs):
         return "X-VIUR-AUTH-User-Password"
 
     class LoginSkel(skeleton.RelSkel):
-        name = EmailBone(descr="E-Mail", required=True, caseSensitive=False, indexed=True)
-        password = PasswordBone(descr="Password", indexed=True, params={"justinput": True}, required=True)
+        name = EmailBone(
+            descr="Username",
```

Comment:
That's still an email address. I cannot set this to a custom fantasy name as I would expect from a _username_. I've always found this forced name bone very strange anyway. With ViUR4 we could think about renaming it to _email_. And _name_ can then be computed from _firstname_, _lastname_ and/or _email_.

```suggestion
            descr="E-Mail",
```

Reply:
Ok yes, I think this came in as I copied it from the `LostPasswordStep1Skel` - anyway, I have further plans in future for this field, for customization.